app/models/data_file.rb

```diff
@@ -13,15 +13,15 @@ def self.search(environment, key)
     result = {}
     HieraData.new(environment.name).files_including(key.name).each do |file|
       hierarchies[file[:hierarchy_name]] ||= Hierarchy.new(
-        environment: environment,
+        environment:,
         name: file[:hierarchy_name],
```

Comment:
Kommt das von Rubocop?

Reply:
yes, this is from rubocop. its seems one does this now this way. see https://dev.to/baweaver/ruby-3-1-shorthand-hash-syntax-first-impressions-19op